cryptopunkscc/astrald
# Upgrade android module

Couple of fixes and upgrades for android module.

* rename module `wrapper` to `node`
* fixes for build scripts (`clean.sh`, `buildGo.sh`, `buildApk.sh`,`installApk.sh`)
* improve android node log screen
* activity for asking write-storage permissins
* refactor and cleanup
* update readme

How to test manually:
1. To preprare for test, remove existing build artifacts by runnging `./clean.sh`.
1. To generate artifacts run `./buildApk.sh`, wait for success and check if `mobile/android/build` dir contains `astral.arr` and `asral-android.apk`
1. To install apk on conntected android device and start astral node activity run `./installApk.sh`. After success android node should be running on device as android service, and listen on tcp sockets.
